### ISSUE TITLE
Show scrollbars on Linux/Windows

### ISSUE THREAD

**Issue by warpdesign** [warpdesign/react-explorer#99]
Current version of React-Explorer uses overlayScrollbars which are hidding big ugly scrollbars on Windows/Linux. Problem scrollbars seem to be hidden and using touchpad to scroll does not seem to make them appear.

Using [SimpleBar](https://grsmto.github.io/simplebar/) could be a good idea.

Also, the overlayScrollbars of Chrome was removed in Chrome 78 so will likely be removed from Electron as well.

**Comment by warpdesign** [comment]
After some experiments it won't be that easy to use SimpleBar (or react-custom-scrollbars) since React-Explorer is using AutoSize+Table (which itself uses Grid) all from React-Virtualized.

Saw hacks hooking into the onScroll event to position the grid but performance really suffers and it's not easy to implement.

Alternative: using css-only solution `overflow: overlay`. For example:

```css
body {
overflow: overlay;
}

 ::-webkit-scrollbar {
  width: 10px;
  height: 10px;
}

::-webkit-scrollbar-thumb {
  background: rgba(90, 90, 90, .3);
  border-radius:5px;
}

::-webkit-scrollbar-track {
  background: rgba(0, 0, 0, 0);
} 
```

**Comment by warpdesign** [comment]
Elements that need to be scrollable:

 - Dialog
 - FileTable
 - Favorites

**Comment by warpdesign** [comment]
Fixed in #110
